static/src/js/app/components/quick-add-button/QuickAddVocabForm.vue

```diff
@@ -244,7 +243,8 @@
         }
 
         // Get headword from database if it isn't already in state
-        if (!Object.hasOwn(this.$store.state.forms, this.headword)) {
+        // Note: Node v14.X does not have a function called Object.hasOwn so converted to hasOwnProperty
```

Comment:
Interesting note about `Object.hasOwn`. According to https://v8.dev/features/object-has-own it's only available in V8 versions >= 9.3, which means [NodeJS 16.x or higher](https://nodejs.org/en/download/releases/).

Reply:
Yea, I only encountered this problem when I tried running the tests and the environment fail when accessing the quickAdd component since I'm locally running on `Node 14.x` but would otherwise pass if switched to a higher Node version. However when the code is ran in docker everything seems to work as expected with `Object.hasOwn`. If we ever upgrade the Node version in the future we can update accordingly.